crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/FullMeansAssessmentMapper.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
+
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.AssessmentDetail;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.FullMeansAssessment;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+class FullMeansAssessmentMapper {
+
+    private final OutgoingsMapper outgoingsMapper = new OutgoingsMapper();
+
+    @NotNull
+    FullMeansAssessment map(
+            uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.OutgoingsDetails crimeApplyOutgoingsDetails) {
+
+        FullMeansAssessment fullMeansAssessment = new FullMeansAssessment();
+
+        if (crimeApplyOutgoingsDetails == null) {
+            return fullMeansAssessment;
+        }
+        List<AssessmentDetail> assessmentDetails = outgoingsMapper.mapOutgoings(crimeApplyOutgoingsDetails.getOutgoings(), crimeApplyOutgoingsDetails.getHousingPaymentType());
```

Comment:
Optional: I would consider extracting out the duplicated method calls into local variables to aid readability.

e.g.
```
        List<Outgoing> outgoings = crimeApplyOutgoingsDetails.getOutgoings();
        Object housingPaymentType = crimeApplyOutgoingsDetails.getHousingPaymentType();

        List<AssessmentDetail> assessmentDetails = outgoingsMapper.mapOutgoings(outgoings, housingPaymentType);
        fullMeansAssessment.setAssessmentDetails(assessmentDetails);
        fullMeansAssessment.setOtherHousingNote(outgoingsMapper.mapOtherHousingFeesNotes(outgoings, housingPaymentType));

```